staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -253,6 +257,18 @@ func (a *MutatingWebhook) Admit(attr admission.Attributes) error {
 	return a.convertor.Convert(versionedAttr.Object, attr.GetObject())
 }
 
+// TODO: factor into a common place along with the validating webhook version.
+func (a *MutatingWebhook) shouldSkipAllHooks(attr admission.Attributes) bool {
+	gvk := attr.GetKind()
+	if gvk.Group == "admissionregistration.k8s.io" && gvk.Kind == "ValidatingWebhookConfiguration" {
+		return true
+	} else if gvk.Group == "admissionregistration.k8s.io" && gvk.Kind == "MutatingWebhookConfiguration" {
+		return true
+	} else {
+		return false
+	}
```

Comment:
optional nit (shorter/prettier/clearer):

```
  if gvk.Group == "admissionregistration.k8s.io" {
    if gvk.Kind == "ValidatingWebhookConfiguration" || gvk.Kind == "MutatingWebhookConfiguration" {
      return true
    }
  }
  return false
```

Maybe this could go in k8s.io/apiserver/pkg/admission/plugin/webhook/config or k8s.io/apiserver/pkg/admission/plugin/webhook/request? It looks like there's already a few packages there that might be appropriate.

Reply:
I think webhook/config is for the plugin config passed to the apiserver at startup, but request could probably work, I'll try that.